elastic/elastic-integration-corpus-generator-tool
# Updating k8s pod and container templates with no host ips and realistic hostnames

TSDB nightly tests demand a more relastic set of data generated. As part of this enhancement we update kubernetes pod and container templates with following:

- Removing ips and mac addresses to simulate the fix we introduced in https://github.com/elastic/integrations/issues/6674
- Adding realistic host and node names. Our template need a set of 90 nodes where the agents will be located. We correlate the node names with host names

## How to test:
1. Checkout this PR
2. Produce pod and container samples:

```bash
./elastic-integration-corpus-generator-tool generate-with-template ./assets/templates/kubernetes.pod/gotext.tpl ./assets/templates/kubernetes.pod/fields.yml -c ./assets/templates/kubernetes.pod/configs.yml -y gotext -t 2

./assets/templates/kubernetes.container/gotext.tpl ./assets/templates/kubernetes.container/fields.yml -c ./assets/templates/kubernetes.container/configs.yml -y gotext -t 2
``` 

Check that host.ips and host.mac fields are removed
Check the node names created and have the same value as hostnames.

## Screenshot
<img width="1296" alt="Screenshot 2023-10-06 at 1 09 03 PM" src="https://github.com/elastic/elastic-integration-corpus-generator-tool/assets/1685415/78d72f40-0a5a-4911-8d6c-0667e37b3361">
